arrays_strings/unique_chars/unique_chars_solution.ipynb

```diff
@@ -278,22 +278,23 @@
   }
  ],
  "metadata": {
+  "celltoolbar": "Edit Metadata",
```

Comment:
Almost there!  Thanks for making the changes.  Just one more...I'm not sure why `unique_chars_solution.ipynb` is part of this linked list PR, I think this was added by accident?